transport/mqtt/mqtt.go

```diff
@@ -142,24 +147,29 @@ func (m *mqtt) init(ctx context.Context, client mqttClient) (err error) {
 	return
 }
 
-func (m *mqtt) onConnect(server string, code byte, err error) {
+func (m *mqtt) onConnect(server string, _code byte, err error) {
```

Comment:
This feels a little weird. The `_` thing isn't something we normally do in Go, and if all we're going to do is transform it into a `Code`, why not accept a `Code` to begin with?

Reply:
The signature has to match `libmqtt.ConnHandler`:
```
type ConnHandler func(server string, code byte, err error)
```

I could name it something else, like `rawCode` instead